variables.tf

```diff
@@ -124,24 +56,18 @@ variable "eks_worker_groups" {
 	}))
 
   default = [{
-    name                          = "worker-group-medium"
+    name                          = "worker-group-medium-b"
     instance_type                 = "t3a.medium"
     additional_userdata           = ""
     root_volume_type              = "gp2"
 
     #autoscaling group section
-    asg_max_size                  = "4"
-    asg_desired_capacity          = "2"
+    asg_max_size                  = "2"
+    asg_desired_capacity          = "1"
```

Comment:
just a question, have you tested that this # of workers will be able to run OML with dask no problem?

Reply:
Yes, it works with one node for each zone.